applications/ecallmgr/src/ecallmgr_registrar.erl

```diff
@@ -1295,16 +1318,17 @@ breakup_contact(Contact) when is_binary(Contact) ->
 breakup_contact(Contact) ->
     breakup_contact(kz_term:to_binary(Contact)).
 
--spec proxy_path(kz_term:api_binary(), kz_term:api_binary(), kz_term:api_integer()) -> binary().
-proxy_path('undefined', 'undefined', 'undefined') -> <<>>;
-proxy_path('undefined', 'undefined', Port) -> proxy_path('undefined', <<>>, Port);
-proxy_path('undefined', IP, 'undefined') -> IP;
-proxy_path('undefined', IP, Port) -> <<IP/binary, ":", (kz_term:to_binary(Port))/binary>>;
-proxy_path(Proxy, _, Port) ->
+-spec proxy_path(kz_term:api_binary(), kz_term:api_binary(), kz_term:api_integer(), kz_term:api_binary()) -> binary().
```

Comment:
@lazedo won't this result in a websocket request between FS and Kamailio?

Reply:
@k-anderson `proxy_path` is only used for displaying